apix_documentation_swagger_v3.yaml

```diff
@@ -204,10 +204,15 @@ info:
         response has been received yet
       - __on_hold__ This status has to be manually set; transaction will not
         be processed until it is changed back to `initialized`
-      - __confirmed__ Payment was successful
+      - __confirmed__ Payment was successful, in the case of check payment,
+        it means that the check was processed for delivery
       - __rejected__ There was a problem with the payment; you can get more
         information by checking the __error_code__ and __error_message__ in
         the transaction’s response.
```

Comment:
We should add a subtitle, to make it clear that the following statuses apply only to checks

Reply:
Maybe `Check-related statuses`?